### PR TITLE
Refactor packages

### LINKED ISSUE #50: Refactor packages
Refactor all person to student. Update all javadoc, data and test cases.

### PR DESCRIPTION
**Pull request by Fui03**
Update all instances of person to student.

Closes #50

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any person obtaining a copy
+Permission is hereby granted, free of charge, to any student obtaining a copy
```
Comment (Reallyeasy1) [inline LICENSE:5]: Why did you change from person to student for this?

File: LICENSE [hunk 2]
```diff
@@ -2,11 +2,11 @@ MIT License
...
 of this software and associated documentation files (the "Software"), to deal
 in the Software without restriction, including without limitation the rights
 to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-copies of the Software, and to permit persons to whom the Software is
+copies of the Software, and to permit students to whom the Software is
```
Comment (Reallyeasy1) [inline LICENSE:9]: same comment above

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 3]
```diff
@@ -167,9 +167,9 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
         // Triggers the saveAddressBook method by executing an add command
         String addCommand = AddCommand.COMMAND_WORD + NAME_DESC_AMY + PHONE_DESC_AMY
                 + EMAIL_DESC_AMY + ADDRESS_DESC_AMY;
-        Person expectedPerson = new PersonBuilder(AMY).withTags().build();
+        Student expectedStudent = new StudentBuilder(AMY).withTags().build();
         ModelManager expectedModel = new ModelManager();
-        expectedModel.addPerson(expectedPerson);
+        expectedModel.addStudent(expectedStudent);
         assertCommandFailure(addCommand, CommandException.class, expectedMessage, expectedModel);
```
Comment (Reallyeasy1) [inline src/test/java/seedu/address/logic/LogicManagerTest.java:173]: lgtm

**Review by Reallyeasy1 — COMMENTED** [review]
LGTM, only 1 minor adjustment

File: LICENSE (hunk 1 above)
Comment (Fui03) [reply LICENSE]: Resolved!

File: LICENSE (hunk 2 above)
Comment (Fui03) [reply LICENSE]: Resolved!
